lisa/trace.py

```diff
@@ -68,6 +68,17 @@ def add_events_deltas(self, df, col_name='delta', inplace=True):
         """
         Store the time between each event in a new dataframe column
 
+                  |        |            |
+                  |        |            |
+                  |        |            |
+            ------+--------+------------+------
+                [n-1]     [n]         [n+1]
+
+        This function assumes that at time [n] the event starts and at [n+1]
+        the event stops, so the formula for the returned value is:
+
+            delta[n] = index[n+1] - index[n]
+
```

Comment:
Sphinx is getting confused by the diagram because it's not in an example field. I'd do something like this:

```
This function [...] the returned value is ::

  <diagram here>

  delta[n] = index[n+1] - index[n]
```

Reply:
Thanks for the tip! I was trying to figure this out